docs/sql/statements/alter-table.rst

```diff
@@ -178,8 +178,8 @@ expression.
 Can be used to open or close the table, respectively. Closing a table prevents
 all operations, except ``ALTER TABLE ... OPEN``, to fail. Operations on closed
 partitions will not produce an exception, but will have no effect. Similarly,
-like ``SELECT`` and ``INSERT`` on partitioned will exclude closed partitions and
-continue working.
+like ``SELECT`` and ``INSERT`` on partitioned will exclude closed partitions
+and continue working.
```

Comment:
Is it just me feeling this or could this whole paragraph be written more concisely?

Reply:
I edited this to the following:

> Can be used to open or close the table.
>
> Closing a table means that all operations, except ``ALTER TABLE ... OPEN``, will fail. Operations that fail will not return an error, but they will have no effect.

I cannot parse the following:

> Similarly, like ``SELECT`` and ``INSERT`` on partitioned tables, will exclude closed partitions and continue working.

I have removed this sentence. but I would like @seut or @mkleen to verify that it's okay to remove this sentence. or, if not, suggest an alternative wording

Reply:
Thank you!

> I cannot parse the following.

Yes, the same thing happened to me as well.

Actually the same with:

> Closing a table means that all operations, except ALTER TABLE ... OPEN, will fail. Operations that fail will not return an error, but they will have no effect.

Wouldn't it be more straightforward - if I actually get this right - to just write:

> Closing a table means that all operations, except ``ALTER TABLE ... OPEN``, will be ignored.

Sorry that I haven't used those features myself, that's why I am a bit clueless about them.

Reply:
@amotl I parsed the first sentence and rewrote it as:

> Closing a table means that all operations, except ``ALTER TABLE ... OPEN``, will fail. Operations that fail will not return an error, but they will have no effect.

Reply:
Thank you. See also @seut's comment at https://github.com/crate/crate/pull/11162#discussion_r599461426.